test/api/groups.coffee

```diff
@@ -13,11 +13,70 @@ describe "Groups", ->
       registerNewUser ->
         User.findByIdAndUpdate user._id,
           $set:
-            "balance": 4
+            "balance": 10
           , (err, _user) ->
             done()
       , true
 
+    it "can create a public guild", (done) ->
+      request.post(baseURL + "/groups").send(
+        name: "TestGroup"
+        type: "guild",
+        privacy: "public"
+      ).end (res) ->
+        expectCode res, 200
+        guild = res.body
+        expect(guild.members.length).to.equal 1
+        expect(guild.leader).to.equal user._id
+        done()
+
+    it "can create a private guild", (done) ->
+      request.post(baseURL + "/groups").send(
+        name: "TestGroup"
+        type: "guild",
+        privacy: "private"
+      ).end (res) ->
+        expectCode res, 200
+        guild = res.body
+        expect(guild.members.length).to.equal 1
+        expect(guild.leader).to.equal user._id
+        done()
+
+    it "prevents user from creating a guild when the user has 0 gems", (done) ->
+      registerNewUser (err, user) ->
```

Comment:
I would use a variable other than `user`, since that's on the global namespace for these tests and it makes it confusing what you're trying to do. `user_with_no_gems` or `_user` would work.